src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -36,6 +36,27 @@
             contentType: 'application/vnd.ibexa.api.ContentTypeFieldDefinitionReorder+json',
         },
     };
+    const inputTypeToPreventSubmit = [
```

Comment:
We have the exact mechanism in `src/bundle/Resources/public/js/scripts/admin.content.edit.js`, so I think we should create a helper for this given this is a few duplicated lines and should be easily extracted.

Reply:
on the one hand, yes, on the other hand, if we have several functional buttons, we should not presume which button the customer wanted to press or if enter was accidentally pressed

Reply:
I would call @NataliaBecla (I either blind, or rest of UX team is missing github accounts) on it.

from my pov, if there is a save/publish button it should be a default one under `enter` action. I do not see a case when someone wants to delete a content type by pressing enter. I am also not sold on the accidental clicks, when if I recall correctly we had multiple issues in the past when some form was non submitable by enter key.